grpc-gateway/client/observeDevices.go

```diff
@@ -77,9 +76,7 @@ func (c *Client) ObserveDevices(ctx context.Context, handler DevicesObservationH
 	sub, err := c.NewDevicesSubscription(ctx, &devicesObservation{
 		h: handler,
 		removeSubscription: func() {
-			if _, err := c.stopObservingDevices(ID.String()); err != nil {
-				handler.Error(fmt.Errorf("failed to stop device('%v') observation: %w", ID.String(), err))
-			}
+			_, _ = c.stopObservingDevices(ID.String())
```

Comment:
Could you add a short comment? Usually ignoring errors in non-testing code is suspicious, so we know why it's done here.